velox/functions/prestosql/window/LeadLag.cpp

```diff
@@ -206,28 +220,28 @@ class LeadLagFunction : public exec::WindowFunction {
       }
     }
 
-    return kNullRow;
+    return kDefaultValueRow;
```

Comment:
@xumingming I believe this is the IGNORE NULLS code path. Since I see a change here, I assume this code path also changed.

Reply:
En, indeed. I checked there is unit test for IGNORE NULLs path: https://github.com/facebookincubator/velox/blob/ead8a4ca3f9f369475559cc437f76a8afac253df/velox/functions/prestosql/window/tests/LeadLagTest.cpp#L391-L421

Reply:
@xumingming Would you update the documentation to clarify the behavior when IGNORE NULLS is on and double check that all use cases with IGNORE NULLS are covered by tests?

Reply:
@xumingming , @mbasmanova : We should enhance the above ignoreNulls test to add a single column with only null values and use that as the column parameter in lead/lag to test the behavior in this change explicitly. Looking in the other tests for this scenario is fragile..